kubernetes/resource_kubernetes_namespace_test.go

```diff
@@ -111,21 +111,23 @@ func TestAccKubernetesNamespace_basic(t *testing.T) {
 	})
 }
 
-func TestAccKubernetesNamespace_invalidLabelValueType(t *testing.T) {
-	nsName := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
-
-	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckKubernetesNamespaceDestroy,
-		Steps: []resource.TestStep{
-			{
-				Config:      testAccKubernetesNamespaceConfig_invalidLabelValueType(nsName),
-				ExpectError: regexp.MustCompile("Expected value to be string"),
-			},
-		},
-	})
-}
+// Test disabled: this seems to no longer be an error in Terraform 0.12
+// TODO: Confirm with core team.
```

Comment:
Still an applicable todo?

Reply:
Good point. I missed this one.
Let me clarify that.

Reply:
Done. Clarified with Core that this not relevant anymore (everything turns into a string automatically).